tests/unit/executor_tests/test_build_tfds.py

```diff
@@ -48,8 +48,8 @@ def test_build_tfds_classification():
     train_run(config_file, expriment_id, recreate=True)
 
     # Check if the dataset was build correctly
-    train_data_num = 3
-    validation_data_num = 2
+    train_data_num = 27
+    validation_data_num = 3
```

Comment:
Is this modification related to this PR?

Reply:
@a-hanamoto 
Yes, this is corresponding to the new chosen dataset that replaces the delta-mark dataset, for classification.